code/modules/religion/rites.dm

```diff
@@ -109,6 +109,30 @@
 	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
 	return TRUE
 
+/datum/religion_rites/upgrade_blessing
+	name = "Receive Blessing"
+	desc = "Receive a blessing from the machine god to further your ascension."
+	ritual_length = 15 SECONDS
+	ritual_invocations = list("With hammer and faith our bodies were forged",
+								"Oh,Machine-god,We are your instruments of faith",
+									"Let your energy overload our iron-forged blows")
+	invoke_msg = "The end of flesh is near!"
+	favor_cost = 2500
+
+/datum/religion_rites/upgrade_blessing/invoke_effect(mob/living/user, atom/movable/religious_tool)
+	..()
+	var/altar_turf = get_turf(religious_tool)
+	var/blessing = pick(
+					/obj/item/organ/cyberimp/arm/surgery,
```

Comment:
```suggestion
					/obj/item/organ/cyberimp/arm/surgery,
					/obj/item/organ/cyberimp/arm/toolset,
```
I'm still pushing for a toolset implant to be added to this list. It's perfectly in-theme and CAN be used to help other people.

Reply:
> I'm kinda unhappy with this sect in general, it doesn't really grab me as a good sect that encourages roleplay as much as it is just gamers gaming for gear

my hope is that by also having implants people who dont wanna become an android also have a reason to go talk to a chaplain im also thinking about adding a few more stuff to the greed god

Reply:
im against this because normally having tools takes up a space in your bag or your belt this would make it too powerful

Reply:
no more than a medhud implant would be

Reply:
a medihud in most scenearios will be used by medical staff which helps other players more then hurts them

Reply:
Toolset arm has halved tool speed on top of being free storage. Let's not